src/ray/core_worker/store_provider/plasma_store_provider.cc

```diff
@@ -24,10 +24,16 @@ namespace ray {
 CoreWorkerPlasmaStoreProvider::CoreWorkerPlasmaStoreProvider(
     const std::string &store_socket,
     const std::shared_ptr<raylet::RayletClient> raylet_client,
-    std::function<Status()> check_signals, std::function<void()> on_store_full)
+    std::function<Status()> check_signals, std::function<void()> on_store_full,
+    std::function<std::string()> current_call_site)
```

Comment:
current_call_site sounds like a value not a callback. Maybe get_current_call_site?